$OpenBSD: patch-src_common_h,v 1.6 2012/04/06 10:14:44 dcoppa Exp $

Remove deprecated slide_away code to unbreak with glib-2.32
(upstream svn revision r2234)

--- src/common.h.orig	Fri Apr  6 08:48:38 2012
+++ src/common.h	Fri Apr  6 08:49:16 2012
@@ -432,9 +432,6 @@ gint tv_width;
 gint tv_height;
 gint tv_fps;
 
-GMutex *slide_away;
-GCond *mplayer_complete_cond;
-
 GThreadPool *retrieve_metadata_pool;
 
 gboolean use_mediakeys;
